src/Components/Common/Plausible.tsx

```diff
@@ -6,23 +6,26 @@ import { useEffect } from "react";
 export default function Plausible() {
   const { site_url, analytics_server_url } = useConfig();
 
-  useLocationChange(() => triggerPageView());
-  useEffect(() => triggerPageView(), []);
+  useLocationChange(() => {
+    plausible("pageview");
+  });
+
+  useEffect(() => {
+    plausible("pageview");
+  }, []);
 
   return (
     <Script
       defer
       data-domain={site_url}
-      src={`${analytics_server_url}/js/script.tagged-events.js`}
+      src={`${analytics_server_url}/js/script.manual.tagged-events.js`}
```

Comment:
Why was this changed? Any references?

Reply:
https://plausible.io/docs/script-extensions

https://github.com/coronasafe/care_fe/commit/304ceb8c4ef1eb86e568eabde572a1e4f4761ea2 Removed the `manual` extension and added `tagged-events` extension. Manual extension is required to override the URL. After that commit, all IDs were not being redacted as manual extension was removed.

https://plausible.io/docs/script-extensions#scriptmanualjs

Reply:
You can add extensions by joining it with `.` for the script.js file